lib/interface.rb

```diff
@@ -0,0 +1,96 @@
+class Interface
+
+  attr_reader :menu_array, :order
+
+  def initialize(filepath)
+    @menu = Menu.new(filepath)
```

Comment:
Can you inject the menu instance so that Interface doesn't have to worry about `filepath`, which is a var it doesn't care about?